storybook/stories/ui-charts/manchetteWithSpaceTimeChart/split-simple.stories.tsx

```diff
@@ -0,0 +1,198 @@
+      const monoLineSize = toPixel - fromPixel;
+      if (!monoLineSize) return;
+
+      ctx.fillStyle = AMBIANT_A10;
+      if (spaceAxis === 'x') {
+        ctx.fillRect(fromPixel, 0, monoLineSize, timeSize);
+      } else {
+        ctx.fillRect(0, fromPixel, timeSize, monoLineSize);
+      }
+    },
+    [position]
+  );
+
+  useDraw('overlay', drawMonoTrackSpace);
+
+  return null;
+};
+
+/**
+ * This component shows how to use the useManchetteWithSpaceTimeChart hook with split points:
+ */
+const SplitManchetteWithSpaceTimeChartWrapper = ({
+  waypoints,
+  projectPathTrainResult,
+  selectedTrain,
+  height = 561,
+  splitPoints = [],
+}: {
+  waypoints: typeof SAMPLE_WAYPOINTS;
+  projectPathTrainResult: typeof SAMPLE_PATHS_DATA;
+  selectedTrain: number;
+  height: number;
+  splitPoints: ManchetteWithSpaceTimeChartProps['splitPoints'];
+  scaleWithZoom: boolean;
+}) => {
+  const manchetteWithSpaceTimeChartRef = useRef<HTMLDivElement>(null);
+  const spaceTimeChartRef = useRef<HTMLDivElement>(null);
+
+  const paths = usePaths(projectPathTrainResult, selectedTrain);
+  const { manchetteProps, spaceTimeChartProps, handleScroll } = useManchetteWithSpaceTimeChart({
+    waypoints,
+    manchetteWithSpaceTimeChartRef,
+    height,
+    spaceTimeChartRef,
+    splitPoints,
+    defaultTimeOrigin: Math.min(...projectPathTrainResult.map((p) => +p.departureTime)),
+  });
+
+  return (
+    <div className="manchette-space-time-chart-wrapper">
+      <div
+        ref={manchetteWithSpaceTimeChartRef}
+        className="manchette flex"
+        style={{ height: `${height}px` }}
+        onScroll={handleScroll}
+      >
+        <Manchette {...manchetteProps} />
+        <div className="space-time-chart-container w-full sticky" ref={spaceTimeChartRef}>
+          <SpaceTimeChart className="inset-0 absolute h-full" {...spaceTimeChartProps}>
+            {paths.map((path) => (
+              <PathLayer key={path.id} path={path} color={path.color} level={path.level} />
+            ))}
+          </SpaceTimeChart>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const meta: Meta<typeof SplitManchetteWithSpaceTimeChartWrapper> = {
+  title: 'Manchette with SpaceTimeChart/Split points',
+  component: SplitManchetteWithSpaceTimeChartWrapper,
+};
+
+export default meta;
+
+export const Default = {
+  args: {
+    waypoints: SAMPLE_WAYPOINTS,
+    projectPathTrainResult: SAMPLE_PATHS_DATA,
+    selectedTrain: 1,
+    splitPoints: [
+      {
+        id: SAMPLE_WAYPOINTS[2].id,
+        position: SAMPLE_WAYPOINTS[2].position,
+        size: 100,
+        payload: { name: SAMPLE_WAYPOINTS[2].name },
+        spaceTimeChartNode: <FlatStep position={SAMPLE_WAYPOINTS[2].position} />,
+        manchetteNode: (
+          <SplitElement
+            size={100}
+            position={SAMPLE_WAYPOINTS[2].position}
+            name={SAMPLE_WAYPOINTS[2].name}
+          />
+        ),
+      },
```

Comment:
could we have some document on how to use split points ?
to explain what the payload does for example,
and why do we need to give both spaceTimeChartNode and a manchetteNode

Reply:
Good point - and recent changes made some of that data obsolete. I'll push something ASAP :+1:

Reply:
I just pushed:
- an introductory comment at the beginning of the story, to clarify the intent
- some comments in the `SplitPoint` declaration
- the suppression of the `payload` property, that is no more used since earlier today

Reply:
thanks !